index.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Dmitrij Krutalianau</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+    <base target="_blank">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<div class="page">
+    <main>
+        <h1>Summary of Dima Krutolianov</h1>
+        <div class="container" id="introduction">
+            <div class="header">
+                <h2>Introduction</h2>
+            </div>
+            <div class="introduction__content">
+                <div class="introduction__text">
+                    <p>Hi there!</p>
+                    <p>My name's Dima Krutolianov aka DimaDK24.</p>
+                </div>
+                <div class="introduction__photo">
+                    <img id="me" src="images/photo-min.jpg" alt="My photo" title="My photo">
+                </div>
+            </div>
+        </div>
+        <div class="container" id="contacts">
+            <p>BTW, you can find me in almost all social networks
+                using this nickname. E.g., <a href="https://vk.com/dimadk24">VK</a>, <a href="https://t.me/dimadk24">Telegram</a>,
+                <a href="https://instagram.com/dimadk24">Instagram</a>, <a href="https://fb.com/dimadk24">FB</a> and of
+                course <a href="https://github.com/dimadk24">GitHub</a>.</p>
+            <p>Recently I created <a href="https://www.linkedin.com/in/dimadk24/">a profile on LinkedIn</a>.</p>
+            <p>If prefer old-school messaging, get in touch with me through <a href="mailto:DimaKrutDK@ya.ru">e-mail</a>.
+            </p>
+            <p>Also, you can <a href="tel:+375298757099">call me: +375 29 875-7099</a>.</p>
+        </div>
+        <div class="container" id="story">
+            <div class="header">
+                <h2>Story</h2>
+            </div>
+            <div class="text">
```

Comment:
I haven't seen any styles appleid to class "text". What is it for?

Reply:
@OlgaPoddubko, I applied this class during making HTML. I supposed I'll need it for styles, but the page was looking good without any styles applied to it. And again I didn't improve it after code jam. It's my fail